src/_experimental.scss

```diff
@@ -28,14 +28,14 @@
 		}
 	}
 
-	// PinkDay color
-	.bg-pnkDy {
-		background-color: #AD005F;
+	// // PinkDay color NA
+	// .bg-pnkDy {
+	// 	background-color: #AD005F;
 
-		&.well.header-rwd {
-			background-color: #570030;
-		}
-	}
+	// 	&.well.header-rwd {
+	// 		background-color: #570030;
```

Comment:
This should not be in comment, If there a removal it should be submited in a different pull request